### PR TITLE
Blog post: call for volunteers to join LPGCs

### PR DESCRIPTION
**Pull request by tobyhodges**
I would love a review from @brynnelliott if you have the time?

### CODE REVIEW

File: _posts/2023/01/2023-01-24-lpgc-call-for-volunteers.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+from every community role. 
+**Maintainers** have a vital role to play, bringing their experience of handling contributions to lessons and interacting 
+with the community on GitHub. 
+The voice of **Instructors** will be essential, to inform discussions based on experience gained from teaching workshops and 
+interacting with Learners. 
+**Workshop Hosts** can speak to the value of the lesson program to their communities, and how Carpentries workshops fit into 
+the bigger picture of training and professional development in their context. 
+**Curriculum Advisors** could provide a vital bridge between the governance of curriculum and the lesson program as a whole.
+If you are reading this, would like to apply, but do not fit any of the roles mentioned above, you should still do so! If 
+you have any questions or concerns, please [reach out to me](mailto:tobyhodges@carpentries.org).
```
Comment (brynnelliott) [inline _posts/2023/01/2023-01-24-lpgc-call-for-volunteers.md]:
```suggestion
you have any questions or concerns, please [contact me](mailto:tobyhodges@carpentries.org).
```

File: _posts/2023/01/2023-01-24-lpgc-call-for-volunteers.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+the committee Chair).
+Taking into account other time that may be taken up by communication and other tasks, you can expect LPGC membership to 
+require a minimum of 3 hours every three months.
+
+## How to Apply
+You can [volunteer to join a Lesson Program Governance Committee by filling out this short form][volunteer-application].
+The call for volunteers will close at [the end of Sunday 5th February 2023 Anywhere on Earth][closing-date]
+After this, volunteer applications will be reviewed by members of the Core Team and Executive Council, as well as other 
+volunteers from the community.
+([If you would like to volunteer to review applications, please get in touch!](mailto:tobyhodges@carpentries.org))
```
Comment (brynnelliott) [inline _posts/2023/01/2023-01-24-lpgc-call-for-volunteers.md]:
```suggestion
(If you would like to volunteer to review applications, please [contact me!](mailto:tobyhodges@carpentries.org))
```

File: _posts/2023/01/2023-01-24-lpgc-call-for-volunteers.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+title: "Calling for Volunteers to join a Lesson Program Governance Committee"
+date: 2023-01-19
+time: "09:00:00"
+tags: ["Governance", "Community", "Data Carpentry", "Library Carpentry", "Software Carpentry"]
+---
+
+The Carpentries has three lesson programs, [Software Carpentry][swc], [Library Carpentry][lc], and [Data Carpentry][dc].
+In November, I published [a post describing the governance structure of each lesson program, and outlining a plan to 
+introduce more community governance in the form of _Lesson Program Governance Committees (LPGCs)_][lpgc-planning-post]. 
+**Today, we open the call for volunteers to form these new committees.**
```
Comment (brynnelliott) [inline _posts/2023/01/2023-01-24-lpgc-call-for-volunteers.md]:
```suggestion
**Today, 19 January we open the call for volunteers to form these new committees.**
```

File: _posts/2023/01/2023-01-24-lpgc-call-for-volunteers.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+## What is the Time Requirement?
+As this is the first time that LPGCs are being established, we have no information about the time required for this role. 
+However, committees are expected to meet quarterly, with a minimum of 1 hour of preparation/follow-up work (probably 2 for 
+the committee Chair).
+Taking into account other time that may be taken up by communication and other tasks, you can expect LPGC membership to 
+require a minimum of 3 hours every three months.
+
+## How to Apply
+You can [volunteer to join a Lesson Program Governance Committee by filling out this short form][volunteer-application].
```
Comment (brynnelliott) [inline _posts/2023/01/2023-01-24-lpgc-call-for-volunteers.md]:
```suggestion
You can volunteer to join a Lesson Program Governance Committee by filling out [this short form][volunteer-application].
```

File: _posts/2023/01/2023-01-24-lpgc-call-for-volunteers.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+authors: ["Toby Hodges"]
+teaser: "Guide the development of Software Carpentry, Library Carpentry, or Data Carpentry"
+title: "Calling for Volunteers to join a Lesson Program Governance Committee"
+date: 2023-01-19
+time: "09:00:00"
+tags: ["Governance", "Community", "Data Carpentry", "Library Carpentry", "Software Carpentry"]
+---
+
+The Carpentries has three lesson programs, [Software Carpentry][swc], [Library Carpentry][lc], and [Data Carpentry][dc].
+In November, I published [a post describing the governance structure of each lesson program, and outlining a plan to
```
Comment (brynnelliott) [inline _posts/2023/01/2023-01-24-lpgc-call-for-volunteers.md]: I would link the word 'post' not the entire sentence.

File: _posts/2023/01/2023-01-24-lpgc-call-for-volunteers.md [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+Especially as these committees are being established for the first time, they will be given a large degree of 
+self-determination: in accordance with [the Lesson Program Governance Policy][lp-policy], the Governance Committees will 
+have the power to decide on their own terms, e.g. the length of member term limits, the method by which new members will be 
+selected (e.g. elected or approved), how they overlap/interact with Curriculum Advisors, request additional powers from the 
+Executive Council, etc.
+
+As with other committees in The Carpentries, each LPGC will include a Chair and Secretary. These officers will hold 
+additional responsibilities alongside those of a regular LPGC member, and will receive additional onboarding to ensure they 
+are well-prepared to take on the roles. The responsibilities associated with these officer roles will be almost identical to 
+[those of the Chair and Secretary of Curriculum Advisory Committees][cac-officers].
```
Comment (brynnelliott) [inline _posts/2023/01/2023-01-24-lpgc-call-for-volunteers.md]:
```suggestion
[the Chair and Secretary of Curriculum Advisory Committees][cac-officers].
```

**Review by kariljordan — APPROVED** [review]
This is very informative. Very clear instructions as well, and it's in line with our overall governance. Looks great!

**Comment by elletjies** [comment]
@tobyhodges are you happy for me to merge?

**Pull request merged**